src/modules/template-documentation/TemplateDocumentation.scss

```diff
@@ -1,7 +1,7 @@
 @import 'styles/palette';
 @import 'styles/mixins';
 
```

Comment:
One of those can be deleted if I remember correctly

Reply:
Done